### PR TITLE
Added Sassdoc

### PR DESCRIPTION
**Pull request by awolkers**
Added support for Sassdoc documentation.
Run `gulp css-sassdoc` to generate sassdocs

### CODE REVIEW

File: tasks/css.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ var sourcemaps   = require('gulp-sourcemaps');
 var autoprefixer = require('gulp-autoprefixer');
 var filter       = require('gulp-filter');
 var browsersync  = require('browser-sync');
+var sassdoc  = require('sassdoc');
```
Comment (branneman) [inline tasks/css.js]: Ouch, this indenting hurts my eyes!

File: config.js [hunk 2]
```diff
@@ -18,7 +18,8 @@ module.exports = {
             globStatic:     base.src + '/static/scss/**/!(_)*.scss',
             globStaticAll:  base.src + '/static/scss/**/*.scss',
             globComponents: base.src + '/components/**/*.scss',
-            dirDist:        base.dist + '/static/css'
+            dirDist:        base.dist + '/static/css',
+            sassdocsDist:   'docs/sassdoc'
```
Comment (branneman) [inline config.js]: Why is there no `base.src` or `base.dist` in front of the string?

File: tasks/css.js [hunk 3]
```diff
@@ -31,3 +32,18 @@ module.exports.watch = function() {
...
+/**
+ * Task: Generate Sassdoc documentation
+ */
+module.exports.sassdoc = function() {
+    var options = {
+        dest: config.paths.css.sassdocsDist
+    };
+
+
+    console.log(options.dest);
```
Comment (branneman) [inline tasks/css.js]: Please remove all the unnecessary newlines and `console.log` call.

**Comment by awolkers** [comment]
Did a code cleanup and changed sassdocs dist folder. When later on in development there will be more docs generated we'll have to discuss the location of the docs output.

File: config.js [hunk 4]
```diff
@@ -18,7 +18,8 @@ module.exports = {
             globStatic:     base.src + '/static/scss/**/!(_)*.scss',
             globStaticAll:  base.src + '/static/scss/**/*.scss',
             globComponents: base.src + '/components/**/*.scss',
-            dirDist:        base.dist + '/static/css'
+            dirDist:        base.dist + '/static/css',
+            sassdocsDist:   base.dist + 'docs/sassdoc'
```
Comment (branneman) [inline config.js]: You're missing a slash, on OSX it currently compiles to `distdocs/sassdoc`, this should be:
```js
base.dist + '/docs/sassdoc'
```

Comment (awolkers) [inline config.js]: code is updated with slash

**Pull request merged**